src/Config.php

```diff
@@ -228,6 +218,22 @@ private static function getTerminusScript()
     }
 
     /**
+     * Ensures that directory paths work in any system
```

Comment:
Hopefully with bash support in Win10 on it's way we will have less cause to care 😀